cmd/lvldb2elastic/config.toml

```diff
@@ -0,0 +1,8 @@
+[general]
+    dbPath = "../../db/testnet-vali"
```

Comment:
this should be renamed

Reply:
it was only for testing (this is my chain ID which I tested with). I would keep the chainID in the path because it would be easier. renamed to `dbPathWithChainID`